### DISCUSSION TITLE
Registerd device

### DISCUSSION THREAD

**Discussion by k2000vi** [rustdesk/rustdesk-server-pro#346] [Q&A]
We have a licence with 500 registered device, but after the update 1.4.1  all more than 200 device will disabled and all new connection is refused.
 
we use the script for delete device and it's ok.
 
But, is it possible to not registered device at each connection ? we want just Unattended connection. i'can' find how to do this

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/discussions/182#discussioncomment-10083053
